spec/dummy/.controlplane/Dockerfile.invalid

```diff
@@ -0,0 +1,7 @@
+# syntax = docker/dockerfile:1
+
+# Make sure RUBY_VERSION matches the Ruby version in .ruby-version and Gemfile
+ARG RUBY_VERSION=3.1.2
+FROM registry.docker.com/library/ruby:$RUBY_VERSION-slim as base
+
+RUN unexistent
```

Comment:
nonexistent

Reply:
Done in e0e3fb62ec632ea71aa631f3fcb01f2a57ffcc41.